README.md

```diff
@@ -30,26 +30,17 @@ interface Square {
 }
 ```
 
-The first step is to generate some code for runtime checks:
-```bash
-`npm bin`/ts-interface-builder foo.ts
-```
+The first step is to obtain a [type suite](#type-suites) (runtime type definitions) for the TypeScript file.
 
-It produces a file like this:
-```typescript
-// foo-ti.js
-import * as t from "ts-interface-checker";
-
-export const Square = t.iface([], {
-  "size": "number",
-  "color": t.opt("string"),
-});
-...
-```
+`ts-interface-builder` offers two ways to do this:
```

Comment:
I saw your other message explaining this change.

I'd like to keep the example here. You are right that it's redundant along with ts-interface-builder's documentation, but the version here is sufficient for a good number of readers, who don't have to get into further details.

How about after this paragraph, adding `### Option 1: Using the CLI` with the body as before (including the command line, the example of a generated file, and how to import it), and then `### Option 2: Using the Babel macro` with the body containing a link to the details, as well as a snippet of code like
```
import { getCheckers } from "ts-interface-builder/macro";
const {Square} = createCheckers('./foo.ts');
Square.check({size: 1}); 
```

`ts-interface-checker` is the more popular repo, so I think it's good if its README is sufficient to get people to something working.